logstash.go

```diff
@@ -54,17 +35,32 @@ func NewLogstashAdapter(route *router.Route) (router.LogAdapter, error) {
 	}
 
 	return &LogstashAdapter{
-		route: route,
-		conn:  conn,
+		route:         route,
+		conn:          conn,
+		containerTags: make(map[string][]string),
 	}, nil
 }
 
+// Get container tags configured with the environment variable LOGSTASH_TAGS
+func GetContainerTags(c *docker.Container, a *LogstashAdapter) []string {
+	var tags []string
+	if val, ok := a.containerTags[c.ID]; ok {
+		tags = val
```

Comment:
More readable to just return val here and skip the else, removing one indentation.